src/main/java/org/bitsofinfo/hazelcast/discovery/docker/swarm/SystemPrintLogger.java

```diff
@@ -8,10 +8,9 @@
 import java.util.logging.Level;
 import java.util.logging.LogRecord;
 
+@SuppressWarnings("squid:S106")
```

Comment:
remove this

Reply:
@SuppressWarnings is already used in this project.

Reply:
yes, but not referencing third party tooling like this which is not used in the project

Reply:
Understood.  will remove as part of the next PR if that's ok.